src/MailtoLink/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
```

Comment:
I was getting an `eslint` error here for some weird reason that I couldn't get rid of

![alt-text](https://media.giphy.com/media/bkKvvzE9PEcTK/giphy.gif)

Reply:
This is bizarre. I'd guess it's because it's a functional component and doesn't explicitly use `React`, but we already have a number of existing components that do that. I'm fine leaving in the ignore rule for now but if this happens much more we should think about a large-scale ignore.